### ISSUE TITLE
Next BN12.X level is incorrect in the bitverse and when first entering BN12

### ISSUE THREAD

**Issue by alainbryden** [danielyxie/bitburner#3520]
Upon hacking WD in BN12 (The Recursion), we are routed to the Bitverse and given the option to enter BN12 again - but the level displayed is the level we have already just beaten.

For example, imagine we have SF 12.9, as seen below:
![image](https://user-images.githubusercontent.com/2285037/164117603-00dbbc4b-4430-4ef1-bcfb-a180de72bf51.png)

If we destroy BN 12.10 (which we are currently in if we've already got SF 12.9) we are presented with the following on the Bitverse screen:
![image](https://user-images.githubusercontent.com/2285037/164117661-b5104937-fa92-4736-adc4-c4088c230f99.png)

i.e. the fact that we have just defeated BN 12.10 and acquired SF 12.10 is not yet reflected.

If we choose to enter the BN, we can see that we are now granted SF12.10 and start with NF level 10:
![image](https://user-images.githubusercontent.com/2285037/164117751-2635ac10-eb5c-4e27-b630-8dc4c36ea436.png)

But alas, the stats page also thinks we are just now *in* BN 12.10 (but we shouldn't be, we should now be in BN 12.11)
![image](https://user-images.githubusercontent.com/2285037/164117824-f6b0b3da-b865-4fc5-9930-9dce551aaa9a.png)

If we save and reload the game, it now correctly detects that we're actually in BN 12.11:
![image](https://user-images.githubusercontent.com/2285037/164117852-0359ccb2-0daf-4c3b-8e4c-f6a31df53421.png)

------

So in short, the BN12.X value displayed in the Bitverse and when first entering a new Bitnode appears to be off-by-one.

**Comment by DavidGrinberg** [comment]
This seems to be true for all BNs. I started BN 3 for the first time recently, decided to restart and it says "enter BN3.2" - 
![image](https://user-images.githubusercontent.com/13791262/164911769-889258ad-7987-4681-b5a5-5b56d8c6fd2b.png)
![image](https://user-images.githubusercontent.com/13791262/164911789-4e96da6f-9f9a-4811-9282-43d12f1bb6b5.png)
